packages/strapi-admin/admin/src/translations/sv.json

```diff
@@ -0,0 +1,157 @@
+  "app.components.InstallPluginPage.plugin.support-us.description": "Stöd oss genom att köpa en Strapi T-shirt. Det gör att vi kan fortsätta arbeta med projektet och försöka ge dig bästa möjliga upplevelse!",
+  "app.components.InstallPluginPage.title": "Marknad - Plugins",
+  "app.components.InstallPluginPopup.downloads": "ladda ner",
+  "app.components.InstallPluginPopup.navLink.avis": "avis",
+  "app.components.InstallPluginPopup.navLink.changelog": "changelog",
+  "app.components.InstallPluginPopup.navLink.description": "Beskrivning",
+  "app.components.InstallPluginPopup.navLink.faq": "faq",
+  "app.components.InstallPluginPopup.navLink.screenshots": "Skärmdumpar",
+  "app.components.InstallPluginPopup.noDescription": "Ingen beskrivning finns tillgänglig",
+  "app.components.LeftMenuFooter.documentation": "Dokumentation",
+  "app.components.LeftMenuFooter.help": "Hjälp",
+  "app.components.LeftMenuFooter.poweredBy": "Drivs av ",
+  "app.components.LeftMenuLinkContainer.configuration": "Konfigurationer",
+  "app.components.LeftMenuLinkContainer.general": "Allmänt",
+  "app.components.LeftMenuLinkContainer.installNewPlugin": "Marknad",
+  "app.components.LeftMenuLinkContainer.listPlugins": "Plugins",
+  "app.components.LeftMenuLinkContainer.noPluginsInstalled": "Inga plugins installerade än",
+  "app.components.LeftMenuLinkContainer.plugins": "Plugins",
+  "app.components.ListPluginsPage.description": "Lista över de installerade plugins i projektet.",
+  "app.components.ListPluginsPage.helmet.title": "Lista plugins",
+  "app.components.ListPluginsPage.title": "Plugins",
+  "app.components.Logout.admin": "Hantera administratörerna",
+  "app.components.Logout.profile": "Profil",
+  "app.components.Logout.logout": "Logga ut",
+  "app.components.NotFoundPage.back": "Tillbaka till startsidan",
+  "app.components.NotFoundPage.description": "Hittades inte",
+  "app.components.Official": "Officiell",
+  "app.components.Onboarding.label.completed": "% klar",
+  "app.components.Onboarding.title": "Kom igång videor",
+  "app.components.PluginCard.Button.label.download": "Ladda ner",
+  "app.components.PluginCard.Button.label.install": "Redan installerad",
+  "app.components.PluginCard.Button.label.support": "Stöd oss",
+  "app.components.PluginCard.compatible": "Kompatibel med din app",
+  "app.components.PluginCard.compatibleCommunity": "Kompatibel med samhället",
+  "app.components.PluginCard.more-details": "Mer detaljer",
+  "app.components.PluginCard.PopUpWarning.install.impossible.autoReload.needed": "AutoReload-funktionen måste vara inaktiverad. Starta din app med `yarn develop`.",
+  "app.components.PluginCard.PopUpWarning.install.impossible.environment": "Av säkerhetsskäl kan ett plugin bara laddas ner i en utvecklingsmiljö.",
+  "app.components.PluginCard.PopUpWarning.install.impossible.confirm": "Jag förstår!",
+  "app.components.PluginCard.PopUpWarning.install.impossible.title": "Att ladda ner är omöjligt",
```

Comment:
This is a bit harsh. "Det är inte möjligt att ladda ner" or something sounds more polite. OR "Nedladdning är inte möjlig"

Reply:
> Nedladdning är inte möjlig

Fixed.